### DISCUSSION TITLE
AWS funded by NIH and NSF

### DISCUSSION THREAD

**Discussion by abhijna** [nih-cfde/training-and-engagement#390] [AWS] [answered]
Can you put AWS billing on NIH or NSF grants?

**Comment by ctb** [comment, marked as answer]
The answer to this largely depends on your institution. I've had no trouble putting AWS billing on either, once it was set up; but how to get it best set up will be institution dependent. Both Michigan State and UC Davis had a few hoops to jump through before we were allowed to set it up; at UCD, I've had good luck with just using purchasing cards at the dept level.
